config/locales/de.yml

```diff
@@ -47,10 +47,9 @@ de:
         remember_me: "Eingeloggt bleiben"
   midnight: "Mitternacht"
   start_page:
-    welcome_to: "Herzlich willkommen im"
+    welcome_to: "Herzlich Willkommen im"
     workshop_portal_hpi_html: "Workshop<div class='row spacer-15'></div>Portal des HPI."
     register_now: "Registrieren"
-    welcome: "Herzlich Willkommen"
```

Comment:
This isn't used anywhere, we now use the one 4 lines above that. I thought it'd get lost if I didn't remove it now